support/cas-server-support-validation/src/main/java/org/apereo/cas/web/view/Cas30ResponseView.java

```diff
@@ -1,6 +1,7 @@
 package org.apereo.cas.web.view;
 
-import org.apache.commons.lang.StringUtils;
```

Comment:
Great! Thank you. 

One other thing you may want to do is merge with master once, and then update the pull request. Right now, you have about 250+ commits in this PR. If you merge with master, the count should go down to one.

Do you know how you might do that? If not, I can help.

As a follow-up note: always do work on topic branches of your own fork. Never on master.